cloudprem/kubernetes.tf

```diff
@@ -109,31 +117,6 @@ resource "kubernetes_config_map" "dozuki_resources" {
     EOF
 
     "rds-ca.pem" = file(local.is_us_gov ? "vendor/rds-ca-${data.aws_region.current.name}-2017-root.pem" : "vendor/rds-ca-2019-root.pem")
-
```

Comment:
did you intend to delete this index.json map? It should be added to, see the s3 or buckets config in this blob for the proper format.

Reply:
We aren't reading it in the app code any longer, so it should/can be gone.